addons/project/static/src/actions/client_actions.js

```diff
@@ -74,21 +96,21 @@ export async function showTemplateFormView(
     { model, recordId, method = "action_create_template_from_project" }
 ) {
     const action = await env.services.orm.call(model, method, [recordId]);
-    await env.services.action.doAction({
-        type: "ir.actions.act_window",
-        res_model: model,
-        views: [[false, "form"]],
-        res_id: action.params.project_id,
-    });
+    await openFormView(env, { resModel: model, resId: action.params.project_id });
     await env.services.action.doAction(action);
 }
 
 // Task → Template Notification
-registry.category("actions").add("project_show_template_notification", (env, action) => {
+registry.category("actions").add("project_show_template_notification", async(env, action) => {
     const params = action.params || {};
+    const { template_res_id, template_res_model } = action.params;
+    await openFormView(env, { resModel: template_res_model, resId: template_res_id });
+    env.services.action.currentController.config.breadcrumbs?.pop()
```

Comment:
why do we need that? 


```suggestion
    env.services.action.currentController.config.breadcrumbs?.pop();
```

Reply:
> why do we need that?

Previously, when converting a task into a task template, the task template view was opened. Now, we are only showing a notification. Therefore, this change is not necessary.